dockerfiles/container-registry-agent/Dockerfile.dev

```diff
@@ -0,0 +1,74 @@
+FROM snyk/ubuntu as base
+
+LABEL maintainer="Snyk Ltd"
+
+USER root
+
+RUN apt-get update && apt-get install -y ca-certificates git
+
+USER node
+WORKDIR /home/node
+
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/node_modules/.bin
+
+ADD dockerfiles/.scripts/metadata.json ./
+ADD dockerfiles/.scripts/package.json ./
+ADD dockerfiles/.scripts/debug.sh ./
+ADD dockerfiles/.scripts/install.sh ./
+
+RUN ./install.sh
+
+# Removing debug@0.7.4 (setheader transitive) to satisfy some scanners still reporting this false positive
+RUN rm -rf /home/node/node_modules/setheader/node_modules/debug
+RUN rm -rf /home/node/node_modules/tough-cookie
+RUN npm i -g tough-cookie@4.1.3
+RUN mv /home/node/.npm-global/lib/node_modules/tough-cookie /home/node/node_modules
+
+FROM node:16-alpine3.16
```

Comment:
Now this shall be updated as well, right?

Reply:
I'm going to rebase against `master` so the changes from https://github.com/snyk/broker/pull/606 are included.